bin/dune

```diff
@@ -1,7 +1,7 @@
 (library
  (name      cli)
  (modules   cli)
- (libraries cmdliner fmt.cli logs.fmt fmt.tty logs.cli))
+ (libraries cmdliner fmt.cli logs.fmt fmt.tty logs.cli mdx))
```

Comment:
is this needed ? Not sur how it worked before

Reply:
I introduced a dependency on `Mdx.Syntax.t` because I provide cmdliner converters for it. Before that there was no dependency.